config/config.go

```diff
@@ -42,6 +44,8 @@ type Configuration struct {
 	AllowUsers []string `yaml:"allow_users"`
 	// 指定哪些人为此系统的管理员，必须指定，否则所有人都是
 	AdminUsers []string `yaml:"admin_users"`
+	// 钉钉机器人在应用信息中的AppSecret，为了校验回调的请求是否合法
+	AppSecret string `yaml:"app_secret"`
```

Comment:
```suggestion
        AppSecret []string `yaml:"app_secret"`
```

🤔感觉 `AppSecret` 改为数组能满足更多场景 

未认证的组织上限为100个人，对于开发者来说，确实可能创多个组织满足超过100人的需求，并使用同一个接口 (🤣目前我被迫创了两个组织)